### ISSUE TITLE
mumble configure error Namespace support is required to build ICU

### ISSUE THREAD

**Issue by TomErnst** [mumble-voip/mumble#5581]
### Description

While running ".\scripts\vcpkg\get_mumble_dependencies.ps1" from a Powershell instance, I got the error message:
> configure: error: Namespace support is required to build ICU.
 

### Steps to reproduce

Install git from origin --> success
Install Python from origin --> success
Install CMake from origin --> success
Clone Mumble at "https://github.com/mumble-voip/mumble" into \Mumble --> success
Enter \Mumble --> success
Update submodules --> success
Install MSVS 2019 Community (with "C++ MFC" incl, "C++ cmake tools") --> success
Find "get_mumble_dependencies.ps1" in "\Mumble\scripts\vcpkg\" --> success
Open a Powershell-Instance --> success
Enter "Set-ExecutionPolicy -Scope CurrentUser RemoteSigned" --> success
Enter ".\scripts\vcpkg\get_mumble_dependencies.ps1" --> ERROR

### Mumble version

https://github.com/mumble-voip/mumble | 2022-02-20

### Mumble component

Both

### OS

Windows

### Reproducible?

Yes

### Additional information

BIG IMPORTANT URGENT PROJECT !! (see below)

Sorry, I've lost the respective Powershell-output.
I Don't want to lose the third day to reproduce it with a third try (Two tries, needed one day each).
Didn't find any solution, except of maybe [https://github.com/emscripten-core/emscripten/issues/8401], where the last answer assumes too much for my skills, otherwise I would have tried it. The other answers didn't fit to my context.
At both tries I additionally tried to fix them in several ways and got the preconditions seemingly ready, but got errors later on, partly again while getting the dependencies, partly yet later on in Visual Studio, which seemed to be consequences of my (probably wrong) workarounds.

My goal is to integrate it into Visual Studio, to be able to debug and navigate in it fluently.

I am an experienced software-architect (since ~50 yrs), mainly in .NET/VS.
I am right now trying to use Mumble as the new Inworld-/IM-voicing system base to include in **OpenSim/Firestorm** (Vivox will cancel it's hitherto service soon).
I am very new to open source, sorry. And I'm German, sorry for my possibly bad language.

Please help, urgently!

**Comment by Krzmbrzl** [comment]
I think the full output of running the script would be helpful. Presumably this error occurs when running vcpkg install ICU, right?

In that case the report should probably be filed against the vcpkg GitHub repo. But let's first have a look at the script output. 

In the meantime, you could also download a pre-built environment from https://dl.mumble.info/build/vcpkg/ which saves you the effort of having to build it yourself. 

EDIT: I just saw that you already filed a report against vcpkg: https://github.com/microsoft/vcpkg/issues/23127

**Comment by Krzmbrzl** [comment]
Are you executing the script in a developer powershell prompt or a regular one? If the latter, it might be worth a try to start it from the x64 developer powershell prompt :thinking:

**Comment by TomErnst** [comment]
> I think the full output of running the script would be helpful. Presumably this error occurs when running vcpkg install ICU, right?
> 
> In that case the report should probably be filed against the vcpkg GitHub repo. But let's first have a look at the script output.
> 
> In the meantime, you could also download a pre-built environment from https://dl.mumble.info/build/vcpkg/ which saves you the effort of having to build it yourself.
> 
> EDIT: I just saw that you already filed a report against vcpkg: [microsoft/vcpkg#23127](https://github.com/microsoft/vcpkg/issues/23127)


Yes, the log in the first issue should help, sorry, didn't remember, I have become already very exhausted with it ;)

How do I have to work with the prebuilt environment?
How and where to install it consistently?
Does it work with VS2019-integration?
At which point of the procedure can I switch to VS2019?
Which settings have to be done there respecting this prebuilt environment and it's possible paths?

I am completely new to "vcpkg", resp. it's possible techniques and boundary conditions.
I am even relative new to CMake and git, always used the MS Team Server and it's version control system hitherto, no multiple platform targets ... at least I can deal with the basics meanwhile, but only with the basics.

**Comment by TomErnst** [comment]
[icu.zip](https://github.com/mumble-voip/mumble/files/8103759/icu.zip)

**Comment by TomErnst** [comment]
Meanwhile I:

1st: copied the contents of "windows-static-1.5.x~2021-07-27~64f88807.x64" into **M:\Mumble\vcpkg\**.

2nd: have set the CMake configuration of VS2019 to
> CMake toolchain file: "**M:/Mumble/vcpkg/scripts/buildsystems/vcpkg.cmake**"
> CMake command arguments: "**-DVCPKG_TARGET_TRIPLET=x64-windows-static-md -Dstatic=ON -DIce_HOME=M:/Mumble/vcpkg/installed/x64-windows-static-md**"

The Visual Studio CMake run gives:

> **CMake Error at cmake/pkg-utils.cmake:87 (message):**
>   **Qt5 component not found: Core		M:\Mumble\cmake/pkg-utils.cmake	87**

**Comment by Krzmbrzl** [comment]
No idea how the VS integration works. I could only help you with setting up a command line cmake build :shrug:

**Comment by TomErnst** [comment]
> No idea how the VS integration works. I could only help you with setting up a command line cmake build 🤷

Should I open a new issue then?

**Comment by Krzmbrzl** [comment]
What for? For getting help with VS integration? I new issue will hardly help. Most of the time I'm the one to process all issues here anyway :shrug:

But I suppose it would be a good idea to start with a command line build and then work towards integrating everything into VS somehow 👀

**Comment by davidebeatrici** [comment]
I regularly use Visual Studio to develop on Windows and I've never encountered this issue.

I would:

1. Delete the entire `M:/Mumble/vcpkg` directory and then extract our environment again.
2. Delete CMake's build directory, to make sure no cache remains.
